perfkitbenchmarker/linux_virtual_machine.py

```diff
@@ -771,6 +771,9 @@ def PrepareVMEnvironment(self):
     if not self._CheckDockerExists():
       self.Install('docker')
     self.InitDocker()
+
+    # Python is needed for RobustRemoteCommands
+    self.Install('python')
```

Comment:
I think the script specifically requires python2 (based off of the shebang in wait_for_command.py) - I'm not sure if there are any python3 incompatibilities. While installing "python" will in most cases install some version of python 2, I'd prefer if the package installation actually specified a python version (like 2.7).